subprojects/matrix-benchmarking-workloads/rhods-notebooks-ux/horreum_lts_store.py

```diff
@@ -20,85 +20,96 @@ def register_lts_metric(cluster_role, metric):
+
     if val_type is datetime.datetime:
         return datetime.datetime.timestamp(val)
+
     elif val_type in [dict, types.SimpleNamespace, common.MatrixEntry, defaultdict, list]:
-        parsed = _decode_ci_items(val)
-        return parsed
+        return _decode_ci_items(val)
+
     elif val_type not in type_skiplist:
         return val
 
 
-def _decode_ci_items(obj):
-    obj_type = type(obj)
-
-    if obj_type == list:
-        return [ _parse_entry(val) for val in obj ]
-    elif obj_type in [types.SimpleNamespace, common.MatrixEntry, dict, defaultdict]:
-        if obj_type in [types.SimpleNamespace, common.MatrixEntry]:
-            obj = vars(obj)
-        res = {}
-        for (key, val) in obj.items():
-            res[key] = _parse_entry(val)
-        return res
-    return obj
-
-
-def _encode_entry(obj):
-    obj_type = type(obj)
-
-    if obj_type == list:
-        return [ _encode_entry(val) for val in obj ]
-    elif obj_type is not dict:
-        return obj
-
-    result = {}    
-    final_type = obj.get("$type", "dict")
-    try:
-        del obj['$type']
-    except Exception:
-        pass
-    
+def _decode_ci_items(src_obj):
+    src_obj_type = type(src_obj)
+
+    if src_obj_type == list:
+        new_obj = [_parse_entry(val) for val in src_obj ]
+
+    elif src_obj_type in [types.SimpleNamespace, common.MatrixEntry, dict, defaultdict]:
+        _src_obj = vars(src_obj)
+            if src_obj_type in [types.SimpleNamespace, common.MatrixEntry] \
+            else src_obj
+
+        new_obj = {key: _parse_entry(val) for key, val in _src_obj.items()}
+
+    else:
+        new_obj = src_obj
+
+    return new_obj
+
+
+def _encode_entry(src_obj):
+    src_obj_type = type(src_obj)
+
+    if src_obj_type == list:
+        return [_encode_entry(val) for val in src_obj]
+    elif src_obj_type is not dict:
+        return src_obj
+
+    result = {}
+
+    final_type = src_obj.get("$type", "dict")
+    try: del src_obj['$type']
+    except Exception: pass
+
     if final_type == "datetime":
-        return datetime.datetime.fromtimestamp(obj.get('value'))
+        return datetime.datetime.fromtimestamp(src_obj.get('value'))
 
-    for (key, val) in obj.items():
+    for key, val in src_obj.items():
         result[key] = _encode_entry(val)
 
     if final_type == "defaultdict":
         return defaultdict(types.SimpleNamespace, result)
+
     elif final_type == "SimpleNamespace":
         return types.SimpleNamespace(**result)
-    return result 
-         
 
-def _encode_json(obj, name, settings):
-    del obj['$schema']
-    data = _encode_entry(obj)
+    return result
+
+
+def _encode_json(src_obj, name, settings):
+    del src_obj['$schema']
+
+    data = _encode_entry(src_obj)
+
     return common.MatrixEntry(import_key=name, processed_settings=settings, **data)
 
 
 def build_lts_payloads() -> dict:
-    for(_, entry) in common.Matrix.processed_map.items():
+    for entry in common.Matrix.processed_map.values():
```

Comment:
no unnecessary parenthesis,
no need for `.items()` when you don't use the key, better use `.values()` in this case